turbo.json

```diff
@@ -6,14 +6,170 @@
       "dependsOn": ["cli#build", "^build"]
     },
 
-    // lint tasks
-    "lint": {},
-    // root lint task runs toml check and prettier
-    // TODO: run prettier in individual workspaces instead of globally.
+    // run prettier check before eslint
+    "lint": {
+      "dependsOn": ["lint:prettier"]
+    },
+
+    // root lint task runs toml check
     "//#lint": {
-      "dependsOn": ["//#check:toml", "//#check:prettier"]
+      "dependsOn": ["//#check:toml"]
     },
-    "//#check:prettier": {},
+
+    // run lint:prettier in all workspaces
+    "lint:prettier": {
+      "inputs": [
```

Comment:
I'm guessing these are all the extensions that prettier will run against. In favor of us keeping our `turbo.json` more readable, I'd prefer if we just added the ones we use. In order to future proof from some bad cache hits we could add all of these patterns to `.prettierignore`.

Reply:
you want to fight it out with @nathanhammond ?
https://github.com/vercel/turbo/pull/3925#discussion_r1115353284